src/condor_starter.V6.1/starter_guidance.cpp

```diff
@@ -604,6 +604,16 @@ mapContentsOfDirectoryInto(
 #endif /* WINDOWS */
 
 
+#define REPLY_WITH_ERROR(cmd,code,subcode) \
+	ClassAd context; \
+	context.InsertAttr( ATTR_COMMAND, cmd ); \
+	context.InsertAttr( ATTR_RESULT, false ); \
+	context.InsertAttr( ATTR_REQUEST_RESULT_CODE, (int)code ); \
+	context.InsertAttr( ATTR_REQUEST_RESULT_SUBCODE, (int)subcode ); \
+	continue_conversation(context); \
+	return true;
+
+
```

Comment:
For macro cleanliness, should the body be enclosed in curly braces, so we don't pollute the context with the new context instance?